packages/wallets/salmon/src/adapter.ts

```diff
@@ -0,0 +1,15 @@
+import { WalletName } from '@solana/wallet-adapter-base';
+import { BaseSalmonWalletAdapter, SalmonWalletAdapterConfig } from './base';
+
+export const SalmonWalletName = 'Salmon' as WalletName<'Salmon'>;
+
+export class SalmonWalletAdapter extends BaseSalmonWalletAdapter {
```

Comment:
Is there any reason to have a `BaseSalmonWalletAdapter`? It looks like it can be collapsed into this subclass. This distinction was only necessary in Sollet so that the web and extension versions could be supported.